src/promptflow-devkit/promptflow/_cli/_pf/_flow.py

```diff
@@ -129,6 +132,43 @@ def add_parser_init_flow(subparsers):
     )
 
 
+def add_parser_save_flow(subparsers):
+    """Add flow save parser to the pf flow subparsers."""
+    epilog = """
+Examples:
+
+# Creating a flex flow folder in a specific path:
+pf flow save --path my-awesome-flow --entry intent:extract_intent --code src
+# Creating a flex flow definition yaml under current folder:
+pf flow save --entry intent:extract_intent
+"""  # noqa: E501
+    add_params = [
+        lambda parser: parser.add_argument(
+            "--entry",
+            type=str,
+            help="The entry to be saved as a flex flow, should be relative to code.",
+            required=True,
+        ),
+        lambda parser: parser.add_argument(
+            "--code", type=str, help="The folder containing the snapshot for the flex flow."
+        ),
+        lambda parser: parser.add_argument(
+            "--path",
+            type=str,
```

Comment:
why cli command lacks some parameters like `sample` compared with sdk interface?

Reply:
some parameters like `signature` and `sample` should be json dictionary, not sure if we should enable them in cli, so cut the parameters for now.